index.html

```diff
@@ -1,12 +1,153 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
 </head>
+
 <body>
+    <nav class="container">
+        <h1>Hasenat</h1>
+        <img src="images/Icon - Menu.png" alt="">
+    </nav>
+    <section class="content-container">
+        <h1>
+            Hi There, <br>I'm Hasenat
+        </h1>
+        <h3>I am a Software Developer</h3>
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded,
+            don’t hestiate to contact me.</p>
+
+        <ul class="social-media">
+            <li><img src="images/github.png" alt=""></li>
+            <li><img src="images/linkedin.png" alt=""></li>
+            <li><img src="images/Vector.png" style="margin-top: 12px;" alt=""></li>
+            <li><img src="images/twitter.png" alt=""></li>
+            <li><img src="images/medium.png" alt=""></li>
+        </ul><br><br>
+        <img class="corner-img" src="images/Group.png" alt="">
+
+    </section>
+
+    <section class="recent-work-container">
+        <h1>My Recent Works</h1>
+        <hr>
```

Comment:
- [ Optional] Please consider replacing the h1 with another h tag as you should have no more than one h1 tag.